openroad.bzl

```diff
@@ -1133,6 +1340,7 @@ def orfs_flow(
       args: dictionary of additional arguments to the flow, automatically assigned to stages
       abstract_stage: string with physical design flow stage name which controls the name of the files generated in _generate_abstract stage
       variant: name of the target variant, added right after the module name
+      mock_area: floating point number, spawns additional _mock_area targets if set
```

Comment:
scale the die width/height by this amount, default no scaling.

Reply:
applied